src/pages/student/profile/index.js

```diff
@@ -1,130 +1,38 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { Helmet } from 'react-helmet'
-import { Tabs, Input, Button, Upload, Form } from 'antd'
-import ProfileCard from 'components/Profile/ProfileCard'
-import General14 from 'components/kit/widgets/General/14'
-import General15 from 'components/kit/widgets/General/15'
+import { useSelector } from 'react-redux'
+import { Alert } from 'antd'
+import ProfilePersonalInfoCard from 'components/Profile/PersonalInformationCard'
+import ProfileAboutCard from 'components/Profile/AboutCard'
 
-const { TabPane } = Tabs
+const UpdateProfileNotice = () => {
+  return (
+    <div className="row mb-3">
+      <div className="col-12">
+        <Alert
+          message="Profile requires update."
+          description="Your profile is incomplete. A complete profile is required to perform actions on Digi Dojo."
+          type="info"
+          showIcon
+        />
+      </div>
+    </div>
+  )
+}
 
 const StudentProfile = () => {
-  const [tabKey, setTabKey] = useState('1')
-
-  const changeTab = key => {
-    setTabKey(key)
-  }
+  const user = useSelector(state => state.user)
 
   return (
     <div>
       <Helmet title="Profile" />
+      {user.requiresProfileUpdate && <UpdateProfileNotice />}
```

Comment:
for all relevant boolean conditions i suggest using the double exclamation mark (!!)

this converts falsey values to false, and truthy values to true

more info: https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript

```suggestion
      {!!user.requiresProfileUpdate && <UpdateProfileNotice />}
```

Reply:
ok!